homeassistant/components/nmbs/sensor.py

```diff
@@ -164,6 +164,7 @@ def update(self) -> None:
 
         if (
             liveboard is None
+            or liveboard == -1
```

Comment:
Based on the link provided, I think you could make this a constant, eg. `API_FAILURE = -1`
```suggestion
            or liveboard == API_FAILURE
```